system.go

```diff
@@ -417,6 +433,11 @@ func (s *System) CreateInterface(name string, iface Interface) error {
 	return nil
 }
 
+// ModifyNetworkInterface updates the attributes of an existing interface
+func (s *System) ModifyNetworkInterface(name string, iface Interface) error {
+	return s.CreateInterface(name, iface)
```

Comment:
If you check out the server then you will see that this is the same operation because the modify interface logic is actually just creating not existing interfaces.